crates/holoom_recipe_execution/coordinator/src/lib.rs

```diff
@@ -60,8 +41,8 @@ fn recv_remote_signal(signal_io: ExternIO) -> ExternResult<()> {
         RemoteHoloomSignal::EvmSignatureRequestRejected { request_id, reason } => {
             emit_signal(LocalHoloomSignal::EvmSignatureRequestRejected { request_id, reason })?
         }
```

Comment:
EvmSignatureRequest signal handling belongs in the crate relating to evm signing offers. I guess there now has to be a signal type per coordinator. (Rather than having a misleading number of variants that will never be encountered.)

Question I'm now interested to know the answer to: Does `recv_remote_signal` only receive signals originating from the same zome? I.e. is it not possible for one coordinator to remote signal another signal?

Reply:
the answer to that question is yes..  remote calls are zome to zome..  in holochain they call them hashed zome spaces that abstract in the DHT